executorlib/base/executor.py

```
@@ -0,0 +1,148 @@
+
+    @property
+    def info(self) -> Optional[dict]:
+        """
+        Get the information about the executor.
+
+        Returns:
+            Optional[dict]: Information about the executor.
+        """
+        if self._process is not None and isinstance(self._process, list):
+            meta_data_dict = self._process[0]._kwargs.copy()
+            if "future_queue" in meta_data_dict.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)